README.md

```diff
@@ -2,24 +2,250 @@
 
 # PMIx Standard
 
-This repository contains the LaTeX source for the PMIx standard document, and discussions regarding the PMIx standard.
+PMIx is an application programming interface (API) specification to provide
+High Performance Computing (HPC) libraries and programming models with
+portable and well defined access to commonly available
+distributed computing system services.
 
-## Contributing to the PMIx Standard
+## Governance
 
-There are two main ways to contribute to the PMIx Standard: Participating in the discussion, and writing sections of the document.
+The PMIx Standard is governed by an Administrative Steering Committee (ASC) comprised of representatives from a consortium of research, academic, and industrial organizations.The ASC shall consist of one representative from each Member organization, as defined below, and shall perform the following functions:
 
-### Participating in the discussion
+* Define the mission of the PMIx Standard.
+* Plan release timelines for versions of the PMIx Standard based on available resources, target features, and target goals of the Member organizations.
+* Vote on all issues requiring community consensus (e.g., acceptance of proposed modifications to the Standard), one vote per Member organization. In addition to voting on proposed modifications, the ASC can be requested to vote on any issue submitted by a Member for consideration.
+* Assemble working groups to address specific areas of interest to the community.
+* Nominate and approve the appointment of individuals to fill the role of Release Manager (per release), as described below.
 
-If you have a question about the PMIx interface or want to raise a question for discussion then please file an Issue against this repository. We are using the Issues to discuss topics, and will link Pull Requests related to that topic to that Issue. This provides the community with a public, archived forum for discussion.
+The ASC is led by two Co-Chairs responsible for organizing and conducting the quarterly meetings. This includes:
 
-### Participatin in the writing
+* Announcing meeting agendas at least one month in advance of the meeting. Any proposal wishing to be considered for the next meeting must be put forward (by assigning the proper label, as described below) prior to that time. The ASC Co-Chairs may allow a proposal to be added to the agenda after this time on a case-by-case basis at their sole discretion.
+* Chairing the quarterly meetings and ensuring an opportunity for all to express their opinion during the meeting
```

Comment:
I like the sentiment for 'ensuring an opportunity for all to express their opinion during the meeting' but I am not sure if it belongs in a governance document. What I mean is, the rest of the document aims to be clear cut about what the rules are but this is a soft requirement that is difficult/impossible to guarantee or measure. Perhaps it would go in a philosophy or code of conduct kind of document?

Reply:
I agree that it isn't a governance rule - just a loose set of definitions of the Co-Chair duties. I'd rather limit the number of documents describing all this so people can just go to one place to get what they need. If it feels too squishy, then I'd suggest we just remove that phrase.